### PR TITLE
sync usage from README

### PR DESCRIPTION
**Pull request by casperdcl**

### CODE REVIEW

File: content/docs/usage.md [hunk 1]
```diff
@@ -1,70 +1,100 @@
 # Using CML
 
-You'll need a GitHub or GitLab account to begin. Users may wish to familiarize
-themselves with [Github Actions](https://help.github.com/en/actions) or
-[GitLab CI/CD](https://about.gitlab.com/stages-devops-lifecycle/continuous-integration/).
+You'll need a GitLab, GitHub, or Bitbucket account to begin. Users may wish to
+familiarize themselves with [Github Actions](https://help.github.com/en/actions)
+or
+[GitLab CI/CD](https://about.gitlab.com/stages-devops-lifecycle/continuous-integration).
```
Comment (0x2b3bfa0) [inline content/docs/usage.md:6]: **You** versus **users**... who wins?

File: content/docs/index.md [hunk 2]
```diff
@@ -33,5 +33,5 @@ specific help. We are very responsive ⚡.
 ✅ Check out our [GitHub repository](https://github.com/iterative/cml) and give
 us a ⭐ if you like the project!
 
-✅ Contribute to DVC [on GitHub](https://github.com/iterative/cml) or help us
```
Comment (0x2b3bfa0) [inline content/docs/index.md:36]: Ouch!

File: content/docs/usage.md [hunk 3]
```diff
@@ -1,70 +1,100 @@
...
           cml-send-comment report.md
 ```
 
-### CML Functions
-
-CML provides a number of helper functions to help package outputs from ML
-workflows, such as numeric data and data vizualizations about model performance,
-into a CML report. The library comes pre-installed on our
+We helpfully provide CML and other useful libraries pre-installed on our
 [custom Docker images](/doc/self-hosted-runners#docker-images). In the above
```
Comment (0x2b3bfa0) [inline content/docs/usage.md:61]: Not for this pull request, but we should proably consider creating a separate section for [...] images, as per https://github.com/iterative/cml.dev/issues/86#issuecomment-900989038; they have more use cases than self-hosted runners.

**Review by 0x2b3bfa0 — CHANGES_REQUESTED** [review]
Looks good! 🎉  Can you please take a look to https://github.com/iterative/cml.dev/pull/99#discussion_r702834191 before merging?

File: content/docs/usage.md (hunk 1 above)
Comment (casperdcl) [reply content/docs/usage.md:6]: lol neither would be my choice. "You" is far too informal, while "users" is too disinterestedly cold. Inconsistency is even worse.

```suggestion
A GitLab, GitHub, or Bitbucket account is required. Familiarity with
[Github Actions](https://help.github.com/en/actions) or
[GitLab CI/CD](https://about.gitlab.com/stages-devops-lifecycle/continuous-integration)
may also be beneficial.
```

File: content/docs/usage.md (hunk 3 above)
Comment (0x2b3bfa0) [reply content/docs/usage.md:61]: Moved to https://github.com/iterative/cml.dev/issues/102

File: content/docs/usage.md [hunk 4]
```diff
@@ -1,70 +1,100 @@
 # Using CML
 
-You'll need a GitHub or GitLab account to begin. Users may wish to familiarize
-themselves with [Github Actions](https://help.github.com/en/actions) or
-[GitLab CI/CD](https://about.gitlab.com/stages-devops-lifecycle/continuous-integration/).
+A GitLab, GitHub, or Bitbucket account is required. Familiarity with
+[Github Actions](https://help.github.com/en/actions) or
+[GitLab CI/CD](https://about.gitlab.com/stages-devops-lifecycle/continuous-integration)
+may also be beneficial.
 Here, will discuss the GitHub use case.
```
Comment (0x2b3bfa0) [inline content/docs/usage.md]: This remark may be a bit outdated, given that we provide separate headings for each platform.

**Review by 0x2b3bfa0 — APPROVED** [review]
:shipit:

**Pull request merged**